db/seeds.rb

```diff
@@ -5,3 +5,32 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+users = [
+  {
+    name: 'User 1'
+  },
+  {
+    name: 'User 2'
+  }
+]
+
+auctions = [
+  {
+    title: 'Throne of Eldraine Booster Box',
+    description: 'New: A brand-new, unused, unopened, undamaged item (including handmade items).',
+    start_bid: 85.00,
+    current_bid: 88.50,
+    end_time: 10.days.from_now.strftime('%FT%TZ')
```

Comment:
Why do you need the `strftime`?

Reply:
@ijpiantanida If I don't put that here, it stays as a different format than the time coming back from postgres, so it's hard to make an assertion. I'm a bit unclear on the best way to test these time formats. If you look at this line and the helper method that I'm using below it in the assertion, do you know if there's a better way of doing this?

Reply:
mmm what I don't get is why do you need it in the seeds file. You're not using these objects for the assertions, right?

Reply:
oh yep, you're totally right